agent/config/config.go

```diff
@@ -462,6 +462,7 @@ func environmentConfig() (Config, error) {
 		TaskCleanupWaitDuration:            parseEnvVariableDuration("ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION"),
 		TaskENIEnabled:                     utils.ParseBool(os.Getenv("ECS_ENABLE_TASK_ENI"), false),
 		TaskIAMRoleEnabled:                 utils.ParseBool(os.Getenv("ECS_ENABLE_TASK_IAM_ROLE"), false),
+		DeleteNonECSImagesEnable:           utils.ParseBool(os.Getenv("ECS_ENABLE_NONECS_IMAGES_REMOVE"), false),
```

Comment:
I would probably name this `ECS_ENABLE_NONECS_IMAGES_CLEANUP`

Reply:
How about `ECS_ENABLE_UNTRACKED_IMAGE_GC` or `ECS_ENABLE_UNTRACKED_IMAGE_CLEANUP`?